cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/spark/SparkCqlEvaluator.java

```diff
@@ -245,6 +251,7 @@ public void run(PrintStream out) throws Exception {
         try (SparkSession spark = sparkBuilder.getOrCreate()) {
             boolean useJava8API = Boolean.valueOf(spark.conf().get("spark.sql.datetime.java8API.enabled"));
             this.typeConverter = new SparkTypeConverter(useJava8API);
+            this.hadoopConfiguration = new SerializableConfiguration(SparkHadoopUtil.get().newConfiguration(SparkContext.getOrCreate().conf()));
```

Comment:
I think you can shorten this a bit by using the existing `SparkSession` like so.
```
this.hadoopConfiguration = new SerializableConfiguration(spark.sparkContext().hadoopConfiguration());
```